nautobot/dcim/api/serializers.py

```diff
@@ -782,7 +750,7 @@ class Meta:
             "url",
             "name",
             "device_type",
-            "device_role",
+            "role",
```

Comment:
This change needs to be documented in the v2 migration docs. Take a look at the format I've used for similar changes in https://github.com/nautobot/nautobot/pull/3012/files#diff-d5e7d13f177913ec08c4f0735b2744d57d1b223ee1eb55fbd577505cb16fea23 and see what you think.